app/store/modules/product/index.js

```diff
@@ -0,0 +1,22 @@
+// store/modules/product/index.js
+
+import product from './mock';
+
+export default {
+  namespaced: true,
+  state: {
+    items: product // demo
```

Comment:
- [ ] JSDoc for property is still missing.

I would recommend that we also add documentation for properties, since most of them will be `null` or boolean initially when we handle real data.

```
const state = {
  /**
    * @type   {object}
    */
   product: productData
}
```